examples/react-webpack/package.json

```diff
@@ -39,9 +39,9 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "resolutions": {
-    "semver": "7.6.0"
+    "semver": "7.5.2"
```

Comment:
Is this a typo? Latest version is 7.6.2

Reply:
For some reason it was still complaining on version 7.6.2 [here](https://github.com/commercetools/commercetools-sdk-typescript/security/dependabot/141)

Reply:
Looking at the report seems that we have multiple resolutions for semver in the lock file

https://github.com/commercetools/commercetools-sdk-typescript/blob/ae315373964a2daba2186a38b795f708b4dc4058/examples/react-webpack/yarn.lock#L4344-L4359

Reply:
I have updated the sermver version to the latest, let see if it fixes it.